cmd/commands/peersrpc_active.go

```diff
@@ -69,6 +69,10 @@ var updateNodeAnnouncementCommand = cli.Command{
 			Name:  "color",
```

Comment:
a suggestion for PR structure: it's a better pattern to do:
1) add new logic internal to the server
2) expose new logic of server via RPC
3) start reading/using new logic from client

a nice way to think about it is: we always want to be able to revert back to any commit and have things make sense. But with this current order, if we reverted to this commit, we would have all these fields that the server is exposing and the client is writing to but it wont actually do anything on the server side which doesnt make sense